public/assets/js/scripts.js

```diff
@@ -292,9 +292,11 @@ $(document).ready(function () {
     const avgPropertyCost = removeCommasAndMakeNumber($('.investment-value-input').data('max-amount'));
     const frequency = $("input[name='investment-frequency']:checked").val();
 
-    let balance = avgPropertyCost - initial;
-    let output;
-    let paymentType;
+    const balance = avgPropertyCost - initial;
+    let output = [];
+    let recommentation = '<h5>Recommended Packages <img src="./assets/img/icons/question-mark.svg" alt="payment"></h5>';
```

Comment:
```suggestion
    let recommendation = '<h5>Recommended Packages <img src="./assets/img/icons/question-mark.svg" alt="payment"></h5>';
```
I think you have a typo above, can you fix it, please?